libs/blocks/path-finder/path-finder.css

```diff
@@ -0,0 +1,91 @@
+:root {
+  /* Colors */
+  --hlx-sk-bg: rgb(48 48 48);
+  --hlx-sk-color: #FFF;
+  --notch-size: 12px;
+}
+
+body {
+  background: var(--hlx-sk-bg);
+  color: var(--hlx-sk-color);
+}
+
+.path-finder form {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 90px 50px;
+  align-items: center;
+  gap: 8px;
+  height: 49px;
+  box-sizing: border-box;
+}
+
+.path-finder button,
+.path-finder input {
+  color: #FFF;
+  display: block;
+  font-family: var(--body-font-family);
+  box-sizing: border-box;
+  background: none;
+}
+
+.path-finder input {
+  height: 49px;
+  font-size: 18px;
+  padding: 4px 12px;
+  border: none;
+  outline: none !important;
+}
+
+.path-finder span {
+  height: 49px;
```

Comment:
I see this `49px` in multiple place, would it justify creating a variable for it?

Reply:
Thanks @overmyheadandbody. I've created a variable.